src/libraries/System.Net.Security/src/System/Net/Security/Pal.OSX/SafeDeleteSslContext.cs

```diff
@@ -141,10 +142,16 @@ protected override void Dispose(bool disposing)
         {
             if (disposing)
             {
-                if (null != _sslContext)
+                SafeSslHandle sslContext = _sslContext;
+                if (null != sslContext)
```

Comment:
```suggestion
                if (_sslContext is object)
```

Reply:
> if (_sslContext is object)

We've avoided that style of null check in dotnet/runtime.  Until you learn the idiom, it's really strange, and it doesn't work in all cases we want to do such non-null checks, e.g. it can't be used with pointers.